components/camel-salesforce/camel-salesforce-maven-plugin/src/test/java/org/apache/camel/maven/CamelSalesforceMojoOutputTest.java

```diff
@@ -77,21 +80,25 @@
     @Test
     public void testProcessDescription() throws Exception {
         final File pkgDir = temp.newFolder();
-
         final GenerateMojo.GeneratorUtility utility = mojo.new GeneratorUtility();
 
         final RestClient client = mockRestClient();
 
         mojo.descriptions = new ObjectDescriptions(client, 0, null, null, null, null, mojo.getLog());
 
-        mojo.processDescription(pkgDir, description, utility);
+        Set<String> sObjectNames = StreamSupport.stream(mojo.descriptions.fetched().spliterator(), false)
+                .map(d -> d.getName())
```

Comment:
If you want to be fancy, method reference can be used here
```suggestion
                .map(SObjectDescription::getName)
```